<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calimero</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="layout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tab_calibrationedit">
        <property name="whatsThis">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_template">
         <attribute name="title">
          <string>Template Editor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="ParameterEditGui" name="widget_4" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Calibrationparameters</string>
         </attribute>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="ParameterGroupEditGui" name="widget_2" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>Compareparameters</string>
         </attribute>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="CompareParameterEditGui" name="widget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab3">
         <attribute name="title">
          <string>Calibrationsettings</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="CalibrationEditGui" name="widget_3" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab4">
         <property name="accessibleName">
          <string/>
         </property>
         <attribute name="title">
          <string>Diagram</string>
         </attribute>
         <attribute name="toolTip">
          <string>TEST HILFE</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="DiagramGui" name="diagram_widget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Start Calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="actionOpen_with_script_import"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveas"/>
    <addaction name="separator"/>
    <addaction name="actionSchlie_en"/>
    <addaction name="actionRun_in_Background"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCalimero_Hilfe"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionsettings"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuTools"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionnew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSchlie_en">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Calimero</string>
   </property>
  </action>
  <action name="actionCalimero_Hilfe">
   <property name="text">
    <string>Calimero Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionsaveas">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionsettings">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionRun_in_Background">
   <property name="text">
    <string>Run in Background</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionOpen_with_script_import">
   <property name="text">
    <string>Open with script import</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompareParameterEditGui</class>
   <extends>QWidget</extends>
   <header>./gui/CompareParameterEditGui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParameterGroupEditGui</class>
   <extends>QWidget</extends>
   <header>./gui/ParameterGroupEditGui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalibrationEditGui</class>
   <extends>QWidget</extends>
   <header>./gui/CalibrationEditGui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DiagramGui</class>
   <extends>QGraphicsView</extends>
   <header>./gui/DiagramGui.h</header>
  </customwidget>
  <customwidget>
   <class>ParameterEditGui</class>
   <extends>QWidget</extends>
   <header>./gui/ParameterEditGui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionSchlie_en</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>destroyed()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tab_calibrationedit</sender>
   <signal>currentChanged(QWidget*)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
